code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -750,6 +780,9 @@
 		/obj/item/flash/cyborg/alien,
 		/obj/item/reagent_containers/spray/alien/stun,
 		/obj/item/reagent_containers/spray/alien/smoke,
+		/obj/item/crowbar/cyborg,
+		/obj/item/gps/cyborg,
+		/obj/item/pinpointer/operative/nad,
```

Comment:
```suggestion
		/obj/item/gps/cyborg
```
This is a normal crew-accessible borg. Shouldn't have the ability to point towards nukies.